sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1223,6 +1223,54 @@ object functions {
   def rint(columnName: String): Column = rint(Column(columnName))
 
   /**
+   * Computes rounded value of the given input.
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def round(e: Double): Column = round(e, 0)
+
+  /**
+   * Computes rounded value of the given input.
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def round(e: Column): Column = round(e.expr, 0)
+
+  /**
+   * Computes rounded value of the given input.
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def round(columnName: String): Column = round(Column(columnName))
+
+  /**
+   * Computes rounded value of the given input.
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def round(e: Double, scale: Int): Column = round(lit(e), scale)
```

Comment:
This probably not necessary, as both of the parameters are literals(constant value).